content/_help/get-started/create-your-account._es.md

```diff
@@ -22,9 +23,7 @@ Siga estos pasos para crear su cuenta de Login.gov.
 
    Como una capa adicional de protección, Login.gov requiere que establezca un segundo método de autenticación para mantener su cuenta segura. A esto se le denomina autenticación de dos factores (2FA).
 
-   Aunque solo requerimos un método secundario de autenticación, le animamos a que agregue dos métodos a su cuenta. De esa forma, tendrá un respaldo si pierde el acceso a su método de autenticación principal.
-
-   Puede escoger entre varias opciones de autenticación.
+Le recomendamos que tenga más de un método de autentificación en su cuenta. Puede elegir entre varias opciones de autentificación. Si selecciona un mensaje de texto o de voz como método de autentificación, tendrá que añadir también otro método de autentificación.
```

Comment:
Looks like the indentation here was important, because this fell out of the numbered list when de-indented.

See: https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.app.cloud.gov/preview/18f/identity-site/cms/help/get-started/create-your-account/es/help/get-started/create-your-account/

```suggestion
   Le recomendamos que tenga más de un método de autentificación en su cuenta. Puede elegir entre varias opciones de autentificación. Si selecciona un mensaje de texto o de voz como método de autentificación, tendrá que añadir también otro método de autentificación.
```

Reply:
Based on what is in production, the indentation was intentional